code/modules/client/preferences.dm

```diff
@@ -547,6 +547,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				else
 					HTML += "<font color=red>[rank]</font></td><td><font color=red><b> \[NON-HUMAN\]</b></font></td></tr>"
 				continue
+			if((rank == "AI") && !(config.allow_ai))
+				HTML += "<font color=red>[rank]</font></td><td><font color=red><b> \[DISABLED\]</b></font></td></tr>"
+				continue

```

Comment:
@bgobandit wait, did you set it to disabled in the config, or with the verb? I remember it worked if it was turned off in the config txt, but not if you used the verb.
@MrStonedOne when I made that issue report, the AI job WAS rollable still if you used the verb. I dunno if it was fixed or not.